api/rtx/src/uvisor_malloc.c

```diff
@@ -0,0 +1,70 @@
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <reent.h>
+
+#include "uvisor-lib/uvisor-lib.h"
+#include "cmsis_os.h"
```

Comment:
Please add a comment that this file must be provided by the host code-base. It's not part of our header files.